lib/share_progress/facebook_variant.rb

```diff
@@ -0,0 +1,23 @@
+require 'share_progress'
+require 'share_progress/variant'
+
+module ShareProgress
+  class FacebookVariant < Variant
+    attr_accessor :facebook_title, :facebook_thumbnail, :facebook_description
+
+    def initialize(facebook_title:, facebook_description:, facebook_thumbnail:, button:)
+      @facebook_title = facebook_title
```

Comment:
The spec indicates optional values with `optional` following the name of the field - none of the variant values have optional after them, so I'm assuming they're not. 

We should probably put length validation on top of the values where it applies, though.